cmd/eval.go

```diff
@@ -23,6 +23,10 @@ var (
 
 func init() {
 	evalCmd.Flags().StringToStringP("replacements", "r", map[string]string{}, "Key value pair of comma delimited values. Example: 'NAMESPACE=foo,bar'")
+
+	var tempSolvedAhead bool
```

Comment:
We can avoid declaring this variable entirely by using the `BoolVar` function instead of `BoolVarP`. The `tempSolvedAhead` variable is only scoped to the init function (and therefore not global), and we don't use it's value anywhere else outside of the `RunE(...)` driver code.